tailwind.config.js

```diff
@@ -74,6 +74,11 @@ module.exports = {
           900: '#f5ed00',
         },
         'dark-ash': '#2D2D2D',
+        'white' : '#ffffff',
+        'light-dark' : '#2D2D2D',
+        'dark-light' : '#F9F5F1',
+        'text-header-dark-color' : '#594C4A',
+        'bottomBanner' : "url('/img/BottomBanner.svg')",
```

Comment:
- On line 77 you can remove it, since tailwind have 'white' utility class by default.
- There's a redundancy dark-ash and light-dark, you can remove yours.
- I think you're not using the 'bottomBanner', pls remove if not using it